tensorflow/core/kernels/data/dataset_utils_test.cc

```diff
@@ -41,6 +42,43 @@ TEST(DatasetUtils, ComputeMoveVector) {
   }
 }
 
+TEST(DatasetUtilsTest, VariantTensorDataRoundtrip) {
+  VariantTensorData data;
+  VariantTensorDataWriter writer(&data);
+  TF_ASSERT_OK(writer.WriteScalar("Int64", 24));
+  Tensor input_tensor(DT_FLOAT, {1});
+  input_tensor.flat<float>()(0) = 2.0f;
+  TF_ASSERT_OK(writer.WriteTensor("Tensor", input_tensor));
+  TF_ASSERT_OK(writer.Flush());
+
+  VariantTensorDataReader reader(&data);
+  EXPECT_OK(reader.status());
```

Comment:
I think should should be TF_EXPECT_OK

Reply:
@jsimsa There is a related item in [RELEASE.md]() as following. Will avoid the use of `EXPECT_OK` in the future.

`
ASSERT_OK / EXPECT_OK macros conflicted with external projects, so they were renamed TF_ASSERT_OK, TF_EXPECT_OK. The existing macros are currently maintained for short-term compatibility but will be removed.
`

Reply:
Good catch, I took care of fixing this during the internal review. The PR should be merged soon with my fix.

Reply:
Great, thanks!